src/jetstream/types.rs

```diff
@@ -14,12 +14,13 @@
 use std::time::Duration;
 
 use crate::header::HeaderMap;
+use crate::rfc3339;
```

Comment:
Is there no crate for this ser/de in the wild?

Reply:
I have just discovered that time has a `serde-well-known` feature for this. I'll rework this to use that.